types/lodash/common/lang.d.ts

```diff
@@ -1114,7 +1114,7 @@ declare module "../index" {
          * @param value The value to check.
          * @return Returns true if value is a plain object, else false.
          */
-        isPlainObject(value?: any): boolean;
+        isPlainObject(value?: any): value is Record<string, unknown>;
```

Comment:
This is not accurate in all cases and could cause issues in certain scenarios (especially in the else block).

Reply:
As far as I know _plain_ object in typecript just the same there is a `Record<string, unknown>` so why can't we check like that?

Reply:
I think here you should check for a specific class type with another is or just throw an error about unknown type (for example)